packages/flutter/lib/src/rendering/semantics.dart

```diff
@@ -735,6 +740,11 @@ class SemanticsOwner extends ChangeNotifier {
     assert(action != null);
     final SemanticsActionHandler handler = _getSemanticsActionHandlerForId(id, action);
     handler?.performAction(action);
+
+    // Default actions if no [handler] was provided.
+    if (handler == null && action == SemanticsAction.showOnScreen && _nodes[id]._showOnScreen != null) {
```

Comment:
The idea is that if a SemanticsActionHandler chooses to implement a custom behavior for the showOnScreen semantics action, the framework should not interfere with that.

Fixed the double null check.